header.css

```diff
@@ -0,0 +1,282 @@
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    position: relative;
+    padding-top: 23px;
+    margin-bottom: 34px;
+    box-sizing: border-box;
+    height: 96px;
+}
+@media screen and (min-width: 1026px) and (max-width: 1171px) {
+    .header {
+        margin-bottom: 18px;
+    }
+}
+@media screen and (max-width: 1023px) {
+    .header {
+        margin-bottom: 0;
+    }
+}
+@media screen and (min-width: 767px) and (max-width: 1023px) {
+    .header {
+        height: 112px;
+    }
+}
+@media screen and (min-width: 375px) and (max-width: 766px) {
+    .header {
+        height: 73px;
+    }
+}
+@media screen and (max-width: 374px) {
+    .header {
+        height: 71px;
+    }
+}
+@media screen and (min-width: 767px) and (max-width: 1023px) {
+    .header {
+        padding-top: 8px; 
+    }
+}
+@media screen and (max-width: 766px) {
+    .header {
+        padding-top: 0; 
+    }
+}
+.header__logo {
+    width: 61px;
+    height: 55px;
+    margin-right: 19px;    
+}
+@media screen and (min-width: 1025px) {
+    .header__logo {
+        width: 55px;
+        margin-right: 45px;
+    }
+}
+@media screen and (max-width: 1023px) {
+    .header__logo {
+        display: none;
+    }
+}
+.header__logo-small {
+    display: none;
+}
+@media screen and (min-width: 767px) and (max-width: 1023px) {
+    .header__logo-small {
+        display: block;
+        width: 156px;
+        height: 40px;
+        margin-left: 43px;
+    }
+}
+@media screen and (max-width: 767px) {
+    .header__logo-small {
+        display: block;
+        width: 109px;
+        height: 22px;
+    }
+}
+@media screen and (min-width: 375px) and (max-width: 767px) {
+    .header__logo-small {
+        margin-left: 45px;
+    }
+}
+@media screen and (max-width: 374px) {
+    .header__logo-small {
+        margin: 0 130px 0 17px;
+    }
+}
+.header__menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-right: 131px;   
+}
+@media screen and (max-width: 1023px) {
+    .header__menu {
+        display: none;
+    }
+}
+@media screen and (min-width: 1025px) {
+    .header__menu {
+        margin-right: auto;
+    }
+}
+.header__menu_link {
+    margin-right: 35px;
+    font: bold 12px/11px 'Gotham Pro';
+    text-transform: uppercase;
+    color: black;
+}
+.header__menu_link:last-child {
+    margin-right: 0;
+}
+.form__search {
+    width: 161px;
+    height: 23px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-right: 18px;
+    border-bottom: 1px solid black;
+    position: relative;
+}
+@media screen and (min-width: 1025px) {
+    .form__search {
+        width: 178px;
+        margin-right: 35px; 
+    }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/final-Axion/pullRequest?prid=5455661)